### PR TITLE
update/fix work area layouts for "1A_BL_BN.csv" and "3-extended.csv"

### PR DESCRIPTION
**Pull request by allan-lu**
1. Split the X-Y coordinates into individual values as shown in [Appendix 19 of the Geosupport UPG](https://nycplanning.github.io/Geosupport-UPG/appendices/appendix19/)
2. In row 29 "Latitude of From Intersection", the "to" value should be 723 not 713. This is a typo carried over from [Appendix 13](https://nycplanning.github.io/Geosupport-UPG/appendices/appendix13/).
3. Plus some other minor fixes that don't have any impact on the outputs.

### CODE REVIEW

File: geosupport/function_info/work_area_layouts/output/3-extended.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name,size,from,to,comment,parent,alt_names,formatter
-List of 4 LGCs for Street 1,8,451,458,On Street,,,2
+List of 4 LGCs for Street 1,8,451,458,�On� Street,,,2
```
Comment (ishiland) [inline geosupport/function_info/work_area_layouts/output/3-extended.csv:2]: are the � characters necessary here?

**Review by ishiland — CHANGES_REQUESTED** [review]
Thanks for the contribution. Getting the tests to pass is outside of the scope of this issue but would be nice to validate these changes in some other way. 

Anyways, only 1 minor comment and I think this is good to go.

File: geosupport/function_info/work_area_layouts/output/3-extended.csv (hunk 1 above)
Comment (allan-lu) [reply geosupport/function_info/work_area_layouts/output/3-extended.csv:2]: No those aren't necessary. They were artifacts from copying over the left and right single quote symbols from the documentation. They can be removed.

File: geosupport/function_info/work_area_layouts/output/1B.csv
Comment (allan-lu) [inline geosupport/function_info/work_area_layouts/output/1B.csv:1]: Also separated the X-Y coordinates in the 1B file.

**Review by ishiland — APPROVED** [review]
Sorry for the delay, I think this looks good. Ill get the package out to pypi later today.

**Pull request merged**